docs/common_links.txt

```diff
@@ -118,4 +121,4 @@
 .. _pytest-astropy: https://github.com/astropy/pytest-astropy
 .. _pytest-doctestplus: https://github.com/astropy/pytest-doctestplus
 .. _pytest-openfiles: https://github.com/astropy/pytest-openfiles
-.. _pytest-remotedata: https://github.com/astropy/pytest-remotedata
\ No newline at end of file
+.. _pytest-remotedata: https://github.com/astropy/pytest-remotedata
```

Comment:
Why is this line in a diff? I can't see any difference here!

Reply:
Oh, I think my editor added a newline at the end of the line where there wasn't one before.  GitHub should show that!